mediaflow_proxy/utils/http_utils.py

```
@@ -1,644 +1,748 @@
+        logger = logging.getLogger(__name__)
+        logger.exception(f"Unexpected error in hls_manifest_proxy: {e}")
+        raise HTTPException(status_code=500, detail=str(e))
 
-    Raises:
-        DownloadError: If the download fails after retries.
+
+async def _handle_hls_with_dlhd_retry(
+    request: Request,
+    hls_params: HLSManifestParams,
+    proxy_headers: ProxyRequestHeaders,
+    dlhd_original_url: str | None
```

Ruff (0.14.6)

Unused function argument: `dlhd_original_url`

(ARG001)